### ISSUE TITLE
Sankaku support?

### ISSUE THREAD

**Issue by ARDEACT** [Bionus/imgbrd-grabber#2262]
This probably isn't the first thread of this, but where the heck is Sankaku?

**Comment by yami-no-tusbas** [comment]
There is no suppot anymore for Sankaku see : https://github.com/Bionus/imgbrd-grabber/issues/2128#issuecomment-697037426 .

**Comment by Tumultuous-RGR-SiM** [comment]
There's a way to use Sankaku. Keep searching for the right support ticket. I forget exactly how I hacked it together, but I wrote out an explanation somewhere in a Sankaku-related support thread some time last year, I think. It involved using a file from an old installation of the program, manually marking it as "do not delete" so the program wouldn't delete the data for Sankaku, and then updating the program.

I do suggest you move on to other boorus, as Sankaku's admin has been throwing fits and running the site into the ground over the last few years and doesn't show signs of improving.

Remember, the first thing you're supposed to do is search the issues section for a few minutes with key terms of your issue to see whether it's been talked about before and what it's status is. The Sankaku question has been coming up a LOT, so I don't think you bothered to search for it. Part of me giving you breadcrumbs here is to encourage you to learn to use the search features on the issues page.
